controller/lib/core/sensors.cpp

```diff
@@ -88,7 +108,21 @@ void zero_sensors() {
       get_raw_sensor_zero_reading(PressureSensors::EXHALATION_PIN);
 }
 
-void sensors_init() {
+void sensors_init(float venturiPortDiameter, float venturiChokeDiameter) {
+  if ((venturiPortDiameter > venturiChokeDiameter) &&
+      venturiChokeDiameter > 0.0f) {
+    venturiPortArea = diameterOfCircleToArea(venturiPortDiameter);
+    venturiChokeArea = diameterOfCircleToArea(venturiChokeDiameter);
```

Comment:
I'm seeing that we take the venturi port/choke diameters as parameters to sensors_init because we want them to be easy to change in the future; is that the motivation?

To me, having them be constants (`PressureSensors::DEFAULT_VENTURI_PORT_DIAM`) already gets us most of the way there.  If we got rid of the params here, we'd also be able to simplify this code, getting rid of the failure mode in the `else` that we're not currently able to handle.  WDYT?

Reply:
With the static assertions it takes care of the concerns I had - will remove.